sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -161,6 +161,14 @@ private[sbtplugin] object ScalaJSPluginInternal {
         val linker = (scalaJSLinker in key).value
         val usesLinkerTag = (usesScalaJSLinkerTag in key).value
 
+        if (!FileVirtualFile.hasExtension(output, ".js")) {
+          throw new MessageOnlyException(
+              s"artifactPath in $key must have '.js' extension")
```

Comment:
Is that really necessary? Can't we just blindly add `.map` at the end of the `path` out `output`, without ensuring it ends in `.js`?

Reply:
Yes we can. I just tried to preserve existing behavior. But probably indeed not necessary.